ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -115,7 +115,7 @@ objects:
 
     containers:
     - name: ignition-exporter
-      image: registry.svc.ci.openshift.org/openshift/origin-v4.0:artifacts
```

Comment:
```
Failed to pull image "registry.ci.openshift.org/openshift/origin-v4.0:artifacts": rpc error: code = Unknown desc = Error reading manifest artifacts in registry.ci.openshift.org/openshift/origin-v4.0: manifest unknown: manifest unknown
```

/hold

```
oc --context api.ci get istag -n openshift origin-v4.0:artifacts
NAME                    DOCKER REF                                                                                                                       UPDATED
origin-v4.0:artifacts   docker-registry.default.svc:5000/openshift/origin-v4.0@sha256:c58657829c3aa7cc5a7295e4fe2a278a5d23c919e31341ac7de04062f666cde4   20 months ago

 oc --context app.ci get istag -n openshift origin-v4.0:artifacts
Error from server (NotFound): imagestreamtags.image.openshift.io "origin-v4.0:artifacts" not found
```

```
oc --context api.ci get is -n openshift origin-v4.0 -o yaml | yq '.metadata'
{
  "annotations": {
    "release.openshift.io/config": "{\n  \"name\":\"4.0.0-0.alpha\",\n  \"to\": \"origin-release\",\n  \"mirrorPrefix\": \"origin-v4.0\",\n  \"expires\":\"72h\",\n  \"publish\":{\n    \"tag\":{\"tagRef\":{\"name\":\"v4.0\"}}\n  },\n  \"verify\":{\n  }\n}",
    "release.openshift.io/source-4.0.0-0.ci": "4.0.0-0.ci-2019-04-10-140345"
  },
```

Found release controller's annotations.
Not sure if we should sync it.

Reply:
```
oc --context app.ci get istag -n openshift origin-v4.0:artifacts
NAME                    IMAGE REFERENCE                                                                                                               UPDATED
origin-v4.0:artifacts   registry.svc.ci.openshift.org/openshift/origin-v4.0@sha256:c58657829c3aa7cc5a7295e4fe2a278a5d23c919e31341ac7de04062f666cde4   2 hours ago
```

Synced.